lib/matplotlib/table.py

```diff
@@ -146,6 +147,67 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class CustomCell(Cell):
+    """
+    A subclass of Cell where the sides may be visibly toggled.
+
+    """
+
+    _edges = 'BRTL'
+    _edge_aliases = {'open':         '',
```

Comment:
trim the extra white space please.

We much not have pep8 compliance on the rest of this file or the test would be failing.